scripts/ci/ci_run_airflow_testing.sh

```diff
@@ -127,16 +127,4 @@ do
     fi
```

Comment:
Actually this whole loop should be removed. It was only a workaround for Travis failures.

We only need to run "run_airflow_testing_in_docker "${@}" that's. it.

Reply:
:D You have a long weekend Jarek, go and enjoy 🚗